### PR TITLE
PLT-975 adding delete to command line tool

### PR DESCRIPTION
**Pull request by coreyhulen**

### CODE REVIEW

File: mattermost.go [hunk 1]
```diff
@@ -30,6 +30,8 @@ var flagCmdCreateUser bool
 var flagCmdAssignRole bool
 var flagCmdVersion bool
 var flagCmdResetPassword bool
+var flagCmdResetPermanentDeleteUser bool
+var flagCmdResetPermanentDeleteTeam bool
```
Comment (hmhealey) [inline mattermost.go]: Shouldn't these be flagCmdPermanentDeleteUser/Team?

**Comment by hmhealey** [comment]
Looks good other than the couple of typos

**Comment by jwilander** [comment]
Do we want to log an audit when a team is deleted as well?

Other than that +1

**Comment by coreyhulen** [comment]
I fixed the typo and added an audit to delete team.

**Pull request merged**